.gitignore

```diff
@@ -0,0 +1,19 @@
+Images/
```

Comment:
I don't think `.gitignore` files should be present in repos.

Reply:
@gwaybio @axiomcura 

Greg, in our last meeting I had asked about if the .gitignore should be ignored or added and you said that .gitignores should be present within the repos. Can you please confirm this?

Reply:
Yes, we include gitignore in analysis repos https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files